configuration/currency/res.currency.csv

```diff
@@ -0,0 +1,5 @@
+id,active
+"base.INR",false
```

Comment:
@nuhuhmutebi , unless I'm wrong, we don't need to disable the INR currency anymore.
This was activated by Bahmni in older version.
Using Odoo 14 with Ozone now should not enable INR.

Reply:
@rbuisson INR is actually set to false, unless you propose that we remove it completely

Reply:
I meant we can remove it from the file altogether and don't have to explicitely disable it.